src/Compilers/CSharp/Portable/Symbols/PropertySymbol.cs

```diff
@@ -332,6 +332,23 @@ public sealed override SymbolKind Kind
             }
         }
 
+        internal int OverloadResolutionPriority
+        {
+            get
+            {
+                if (!CanHaveOverloadResolutionPriority)
+                {
+                    return 0;
```

Comment:
Is this reachable?

Reply:
Not sure what you mean. This is an internal API that may eventually be surfaced as a public API.

Reply:
I'm just asking to make sure my understanding is correct, whether this branch is reachable.

Reply:
What was the resolution?

Reply:
Resolved offline: this is only reachable by tests calling the API directly